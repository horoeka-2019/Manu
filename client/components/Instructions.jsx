import React from 'react'
import { Link } from 'react-router-dom'
import BirdHeader from './BirdHeader'
import MainFooter from './MainFooter'
import BackBtn from './BackBtn'

const Instructions = () => (
    <>
      <BirdHeader />
        <p className="instructions"> He Pokai-Manu is a treasure-hunt game for birds in New Zelands. We aim to help younger Kiwis learn more about our bird life in an exciting way, while getting them outside and interested in nature. </p>
        <br></br>
        <p className="instructions">To play, get outside and get bird spotting! Each time you spot a bird that is listed, go into the bird profile and push 'POKAI'. Your found bird will then be added to your inventory of birds.</p>
        <br></br>
        <p className="instructions">How many can you find?</p>

        <Link to="/">
          <BackBtn/>
        </Link>
        <MainFooter />
    </>
)

export default Instructions
