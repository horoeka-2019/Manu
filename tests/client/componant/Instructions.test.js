import React from 'react'
import { mount } from 'enzyme'

import Instructions from '../../../client/components/Instructions'

describe('<Instructions /> component test', () => {
  it('renders the instructions correctly', () => {
    // Arrange
    const expected = 'He Pokai-Manu is a treasure-hunt game for birds in New Zelands. We aim to help younger Kiwis learn more about our bird life in an exciting way, while getting them outside and interested in nature.'

    // Act
    const wrapper = mount(<Instructions />)
    const actual = wrapper.text()
    // Assert
    expect(actual).toContain(expected)
  })
})