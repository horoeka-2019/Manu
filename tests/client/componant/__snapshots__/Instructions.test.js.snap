// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Instructions /> component test render unchanged from previous snapshot 1`] = `
<Instructions>
  <BirdHeader>
    <Menu
      borderless={true}
      className="centerHeader"
      fixed="top"
      inverted={true}
      style={
        Object {
          "backgroundColor": "#F3A712",
        }
      }
    >
      <div
        className="ui borderless inverted top fixed centerHeader menu"
        style={
          Object {
            "backgroundColor": "#F3A712",
          }
        }
      >
        <Container>
          <div
            className="ui container"
          >
            <MenuItem>
              <div
                className="item"
                onClick={[Function]}
              >
                He Pokai Manu
              </div>
            </MenuItem>
          </div>
        </Container>
      </div>
    </Menu>
  </BirdHeader>
  <p
    className="instructions"
  >
     He Pokai-Manu is a treasure-hunt game for birds in New Zelands. We aim to help younger Kiwis learn more about our bird life in an exciting way, while getting them outside and interested in nature. 
  </p>
  <br />
  <p
    className="instructions"
  >
    To play, get outside and get bird spotting! Each time you spot a bird that is listed, go into the bird profile and push 'POKAI'. Your found bird will then be added to your inventory of birds.
  </p>
  <br />
  <p
    className="instructions"
  >
    How many can you find?
  </p>
  <HashRouter>
    <Router
      history={
        Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <Link
        to="/"
      >
        <LinkAnchor
          href="#/"
          navigate={[Function]}
        >
          <a
            href="#/"
            onClick={[Function]}
          >
            <BackBtn>
              <Button
                as="button"
                className="backBtnStyle"
                size="big"
              >
                <Ref
                  innerRef={
                    Object {
                      "current": <button
                        class="ui big button backBtnStyle"
                      >
                        Back
                      </button>,
                    }
                  }
                >
                  <RefFindNode
                    innerRef={
                      Object {
                        "current": <button
                          class="ui big button backBtnStyle"
                        >
                          Back
                        </button>,
                      }
                    }
                  >
                    <button
                      className="ui big button backBtnStyle"
                      onClick={[Function]}
                    >
                      Back
                    </button>
                  </RefFindNode>
                </Ref>
              </Button>
            </BackBtn>
          </a>
        </LinkAnchor>
      </Link>
    </Router>
  </HashRouter>
  <MainFooter>
    <Menu
      fixed="bottom"
      inverted={true}
      style={
        Object {
          "backgroundColor": "#F3A712",
        }
      }
    >
      <div
        className="ui inverted bottom fixed menu"
        style={
          Object {
            "backgroundColor": "#F3A712",
          }
        }
      >
        <Container
          className="footer"
        >
          <div
            className="ui container footer"
          >
            <MenuItem>
              <div
                className="item"
                onClick={[Function]}
              >
                You have found 6 birds!
                <br />
                Can you find them all?
              </div>
            </MenuItem>
          </div>
        </Container>
      </div>
    </Menu>
  </MainFooter>
</Instructions>
`;
